documentation/src/main/asciidoc/topics/con_shared_cache_stores.adoc

```diff
@@ -0,0 +1,37 @@
+[id='shared-cache-stores_{context}']
+= Shared cache stores
+
+{brandname} cache stores can be local to a given node or shared across all nodes in the cluster.
+
+Configuring cache stores as shared, `shared`, or local only, `shared(false)`,  impacts the rest of your persistent storage configuration.
+
+[IMPORTANT]
+====
+Never configure shared cache stores to fetch state or purge on startup.
```

Comment:
We actually don't allow shared stores to have purge on startup.

Reply:
I've reworded this section to tone down the important note and add a little more detail. Even if we don't allow it I think it's good to advise users not to combine configuration that is going to lead to bad things. Maybe by stating it in the docs it would explain the exception or help users avoid hitting it in the first place.